### DISCUSSION TITLE
Netbox SAML Azure AD Permission

### DISCUSSION THREAD

**Discussion by xonacs** [netbox-community/netbox#9598] [Q&A] [answered]
Hi

So I got Netbox to sign in via Azure AD -> however users get created without any rights. Any guide on how to get netbox to assign rights to users (read groups ?)

Thanks!

**Comment by ryanmerolle** [comment, marked as answer]
I think thats a byproduct of django's social auth project.  Group memebership does not get pulled down like you get from LDAP.  There are ways to do it with the package by putting in a middleware script, but I have not tried it yet personally.
